CHANGES.rst

```diff
@@ -11,7 +11,8 @@ Unreleased
     where the length is invalid, instead of raising an ``AssertionError``. :issue:`2531`
 -   Address remaining ``ResourceWarning`` related to the socket used by ``run_simple``.
     Remove ``prepare_socket``, which now happens when creating the server. :issue:`2421`
-
+-   Update pre-existing headers for multipart/form-data requests in pytest environment
```

Comment:
This changelog mentions "pytest environment" instead of the "Werkzeug test client".